src/routes/project_creation.rs

```diff
@@ -765,3 +785,47 @@ pub fn get_image_content_type(extension: &str) -> Option<&'static str> {
         None
     }
 }
+
+//TODO: In order to ensure readability, only the first error is printed, this may need to be expanded on in the future!
+pub fn validation_errors_to_string(errors: ValidationErrors, adder: Option<String>) -> String {
```

Comment:
Same thing, move it to an utility class, as we may also need this for versions and other things in the future.